svelte/Makefile

```diff
@@ -1,4 +1,6 @@
-SHELL := bash
+SHELL := /bin/bash
+ECHOCMD := /bin/echo -e
```

Comment:
What is the purpose of this?

Reply:
(the ECHOCMD specifically)

Reply:
I though it was a special variable like SHELL, but I was mistaken: https://stackoverflow.com/questions/60387684/how-to-make-the-makefile-echos-to-use-bin-echo-e

I though I had removed them all, but I just see there is still two of them remaining. I am going to remove them right now.

Reply:
I though `ECHOCMD ` was a special variable like `SHELL`, but I was mistaken: https://stackoverflow.com/questions/60387684/how-to-make-the-makefile-echos-to-use-bin-echo-e

I though I had removed all `ECHOCMD`, but I just see there is still two of them remaining. I am going to remove them right now.

The purpose of `SHELL : /bin/bash` is to force the Makefile to not take any `bash` binary on the Windows system path, which just has the name `bash`. On any Windows 10, you can have installed the (https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux), which put a binary named `bash` on the system path. And a `SHELL` variable just like `SHELL := bash` can take that bash to run the make shell commands. The problem is that running that bash is going run a entire Linux subsystem (Ubuntu 18.04) which has by default `python 2.7`. However, even if we take the right `python3` from the `WSL`, it will not work because Anki is graphical application, and this is not supported by the `WSL` yet.

Reply:
I though `ECHOCMD ` was a special variable like `SHELL`, but I was mistaken: https://stackoverflow.com/questions/60387684/how-to-make-the-makefile-echos-to-use-bin-echo-e

I though I had removed all `ECHOCMD`, but I just see there is still two of them remaining. I am going to remove them right now.

The purpose of `SHELL : /bin/bash` is to force the Makefile to not take any `bash` binary on the Windows system path, which just has the name `bash`. On any Windows 10, you can have installed the (https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux), which put a binary named `bash` on the system path. And a `SHELL` variable just like `SHELL := bash` can take that bash to run the make shell commands. The problem is that running that bash is going run a entire Linux subsystem (Ubuntu 18.04) which has by default `python 2.7`. However, even if we take the right `python3` from the `WSL`, it will not work because Anki is graphical application, and this is not supported by the `WSL` yet.